rfc-165-add-api-endpoints-to-transition.md

```diff
@@ -1,3 +1,12 @@
+---
+status: to_be_deleted
+implementation: not_planned
+status_last_reviewed: 2024-03-04
+status_notes: >
+  People aren't clear whether the value of doing this outweighs the cost of implementing it.
```

Comment:
I'm still not 100% on this, I think i'd still like to at least spike it but I'll have a chat with the team :)